bip-0003.md

```diff
@@ -0,0 +1,708 @@
+remain Complete instead. A BIP can also be moved to Closed by the community if it has had Complete status for at least one year,
+there is no evidence of it being in active use, and its authors either do not object or fail to respond; unless a new author
+volunteers within four weeks.
+
+##### Deployed ↦ Closed
+
+A BIP may evolve from Deployed to Closed when it is no longer in active use. Any community member may initiate this
+Status update by announcing it to the mailing list, and proceed if no objections have been raised for four weeks.
+
+##### Closed ↦ Draft
+
+The Closed status is generally intended to be a final status for BIPs,
+and if BIP authors decide to make another attempt at a previously Closed BIP, it is generally recommended to create a new
+proposal. (Obviously, the authors may borrow any amount of inspiration or actual text from any prior BIPs as licensing
+permits.) The authors should take special care to address the issues that caused the prior attempt’s abandonment. Even
+if the prior attempt had been assigned a number, the new BIP will generally be assigned a distinct number. However, if it is
+obvious that the new attempt directly continues work on the same idea, it may be reasonable to return the
+Closed BIP to Draft status.
+
+### Changelog
+
+To help implementers understand updates to a BIP, any changes after it has reached Complete must be tracked with version,
+date, and description in a Changelog section sorted by most recent version first. The version number is inspired by semantic versioning (MAJOR.MINOR.PATCH).
+The MAJOR version is incremented if changes to the BIP’s Specification are introduced that are incompatible with prior
+versions (which should be rare after a BIP is Complete, and only happen in well-grounded exceptional cases to a BIP that
+is Deployed). The MINOR version is incremented whenever the specification of the BIP is changed or extended in a
+backward-compatible way. The PATCH version is incremented for other changes to the BIP that are noteworthy (bug fixes,
+test vectors, important clarifications, etc.). Version 1.0.0 is used to label the promotion to
+Complete. A Changelog section may be introduced during the Draft phase to record significant changes (using versions 0.x.y).
+
+Example:
+
+> __Changelog__
+>
+> * __2.0.0__ (2025-01-22):
+>     * Introduce a breaking change in the specification to fix a bug.
+> * __1.1.0__ (2025-01-17):
+>     * Add a backward compatible extension to the BIP.
+> * __1.0.1__ (2025-01-15):
+>     * Clarify an edge case and add corresponding test vectors.
+> * __1.0.0__ (2025-01-14):
+>     * Complete planned work on the BIP.
+
+After a BIP receives a Changelog, the
+Preamble must indicate the latest version in the Version header. The Changelog highlights revisions to BIPs to human readers. A single
+BIP shall not recommend more than one variant of an idea at the same time. An an updated or
```

Comment:
double "an"

```suggestion
BIP shall not recommend more than one variant of an idea at the same time. An updated or
```

Maybe also `s/An updated or competing variant/A different or competing/`:

- BIPs may have bugfixes or changes (cf top example in Changelog section), and;

- `Process BIPs are living documents that do not ossify and may be modified indefinitely.`